ui/app/AppLayouts/Profile/controls/SettingsList.qml

```diff
@@ -40,20 +40,13 @@ StatusListView {
         title: model.text
         asset.name: model.icon
         selected: root.currenctSubsection === model.subsection
-        highlighted: !!betaTagLoader.item && betaTagLoader.item.hovered
```

Comment:
Was only needed because we placed the Loader on top of the item; now the `highlighted` prop works fine without this workaround